crates/bevy_ecs/src/system/builder.rs

```diff
@@ -386,7 +386,7 @@ mod tests {
             .build_state(&mut world)
             .build_system(local_system);
 
-        let result = world.run_system_once(system);
+        let result = world.run_system_once(system).unwrap();
```

Comment:
Nit: these tests will be clearer if we rename `result` to `output` or something. It's a bit confusing to call the output of unwrapping a `Result` `result` 😜

Reply:
Ah, didn't consider that, just mindlessly unwrapped, good catch